src/app/containers/AssignmentDutyEditForm.tsx

```diff
@@ -39,10 +43,19 @@ export interface AssignmentDutyEditFormProps extends AssignmentDutyFormProps {
 const mapStateToProps = (state: RootState, props: AssignmentDutyEditFormProps) => {
     const initialAssignmentDuty = getAssignmentDuty(props.id)(state);
     if (initialAssignmentDuty) {
-        const { title }: Assignment = getAssignment(initialAssignmentDuty.assignmentId)(state);
+        const initialAssignment: Assignment = getAssignment(initialAssignmentDuty.assignmentId)(state);
         return {
-            initialValues: initialAssignmentDuty, 
-            assignmentTitle: title
+            initialValues: {
+                ...initialAssignmentDuty,
+                timeRange: {
+                    startTime: moment(initialAssignmentDuty.startDateTime).toISOString(), 
```

Comment:
Eventually I think we'll be able to assume that startTime endTime are ISOString representation of dates/times